rdf-delta-server-local/src/main/java/org/seaborne/delta/server/local/patchstores/zk/PatchStoreZk.java

```diff
@@ -196,25 +163,24 @@ protected void sync() {
         try {
             List<String> logNames = activeLogs();
             //FmtLog.debug(LOGZK, "[%s] Store sync : %s", instance, logNames);
-            updateLogChanges(logNames, false);
+            updateLogChanges(logNames);
         } catch (Exception ex) {
             FmtLog.warn(LOGZK, "Exception in sync: "+ex.getClass().getSimpleName()+" "+ex.getMessage());
         }
     }
 
     @Override
     protected void initialize(LocalServerConfig config) {
-        if ( client == null ) {
+        if ( zk == null ) {
             FmtLog.warn(LOGZK, "[%s] No Curator client", instance);
             return;
         }
-        connectToZookeeper();
 
-        boolean isEmpty = zkCalc(()->client.checkExists().forPath(ZkConst.pRoot)==null);
+        boolean isEmpty = !this.zk.pathExists(ZkConst.pRoot);
```

Comment:
Race potential here. ZK has test-and-set or compare-and-swap operations, doesn't it?